tensorflow/lite/tools/make/Makefile

```diff
@@ -101,6 +101,7 @@ $(wildcard tensorflow/lite/*.c) \
 $(wildcard tensorflow/lite/c/*.c) \
 $(wildcard tensorflow/lite/core/*.cc) \
 $(wildcard tensorflow/lite/core/api/*.cc) \
+$(wildcard tensorflow/lite/experimental/resource/*.cc) \
 $(wildcard tensorflow/lite/experimental/resource_variable/*.cc) \
```

Comment:
Actually resource_variable was renamed to resource. We'd better remove this.

Reply:
Done.